server.py

```diff
@@ -44,5 +49,21 @@ async def get_garage_data():
     return garage_data
 
 
+@app.get("/logging-levels")
+def logging_levels():
+    logging.error("error message")
+    logging.warning("warning message")
+    logging.info("info message")
+    logging.debug("debug message")
```

Comment:
this is also from example we dont need this endpoint

Reply:
ok i deleted both parts